fastlane/docs/README.md

```diff
@@ -54,6 +64,18 @@ after_all do |lane|
 end
 ```
 
+### `after_each` block
+
+This block will get executed *after* each lane switch (and before returning to a parent lane). It supports the same actions as lanes.
```

Comment:
If we have
```
lane :deploy do
  sign_ipa
  upload_ipa
end
```

It would run after `sign_ipa` but before returning to `deploy`, again after `upload_ipa` and before returning to `deploy` and then finally after `deploy`